packages/runtime/src/runtime.ts

```diff
@@ -37,7 +36,7 @@ export class Runtime {
     // runtime store
     this.store = {
       accounts: new Map<string, StoreAccountI>(), // string represents account address
-      globalApps: new Map<number, GlobalAppsData>(), // number represents appId
+      globalApps: new Map<number, string>(), // number represents appId
```

Comment:
what `string` represents? Do we have a type for this ? is there a place where this is documented?

Reply:
It represents the account address (which is a string). Adding doc comment in `/types` and here.